src/extension/extension.ts

```diff
@@ -61,6 +62,28 @@ export function activate(context: vscode.ExtensionContext) {
   console.info("Extension is successfully setup.");
 }
 
+function handlePotentialErrorOnKaotoBackendStart(backendProcess: child_process.SpawnSyncReturns<Buffer>, kaotoBackendOutputChannel: vscode.OutputChannel) {
+	if (backendProcess.error !== undefined) {
```

Comment:
Not important for this PR, but for future reference you can also do this as a shorthand, and for all the other checks below: `if(backendProcess.error)` and it will  evaluate to `false` if `backendProcess.error` is: `undefined`, `null`, `NaN`, `false`, `0`, `empty`, or `""` if it's a string

Reply:
oh you didn't have to make the changes, it was just a tip, but thank you :)